docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3"
 
 services:
   postgres:
-    image: postgres:9.5
+    image: postgres:12.0-alpine
```

Comment:
I'd suggest to run from the `12-alpine` tag to also profit from minor releases.

AFAIK database upgrades are only needed for major releases, so this service could attach to the whole 12 mainline.